web/.npmignore

```diff
@@ -0,0 +1,5 @@
+index.html
+server.py
+wasmLoader.js
+rust-toolchain
+.cargo/config
```

Comment:
I'm not entirely sure, but I think npmignore is not recursive like gitignore is, so this file would have to be moved up to the root dir of this repo

Reply:
Yes, good catch. I see that I also had one in the root. I've removed the duplication.